scripts/governance-notifier.ts

```diff
@@ -0,0 +1,101 @@
+import { PublicKey } from '@solana/web3.js'
+import axios from 'axios'
+import { RealmInfo } from '../@types/types'
+import { getAccountTypes, Governance, Proposal } from '../models/accounts'
+import { getGovernanceAccounts, pubkeyFilter } from '../models/api'
+import { ParsedAccount } from '../models/serialisation'
+import { ENDPOINTS } from '../stores/useWalletStore'
+
+global.fetch = require('node-fetch')
```

Comment:
![image](https://user-images.githubusercontent.com/144124/130091853-227db182-5842-4f68-b79c-1898378f041a.png)

Reply:
I would like to re-use the code from the governance-ui, hence this ugly statement 😅